ibm/mas_devops/roles/appconnect_install/tasks/main.yml

```diff
@@ -0,0 +1,188 @@
+---
+
+# 1. Check for required facts
+# -----------------------------------------------------------------------------
+- name: "Fail if AppConnect entitlement username has not been provided"
```

Comment:
Please use asserts instead of fail/when for new code ... see https://github.com/ibm-mas/ansible-devops/issues/124 for more information